keyboards/kumo/keymaps/LEdiodes/keymap.c

```diff
@@ -0,0 +1,212 @@
+// Normal + arrow layout for MiniVan Kumo
+// Created by LEdiodes on 5-2-19
+
+#include "kumo.h"
```

Comment:
Could you please change this to:

```suggestion
#include QMK_KEYBOARD_H
```